collector/command.go

```diff
@@ -79,7 +90,7 @@ func execVtyshCommand(args ...string) ([]byte, error) {
 
 	err := cmd.Run()
 	if err != nil {
-		return nil, fmt.Errorf("%s: %s", err, strings.Replace(stderr.String(), "\n", " ", -1))
+		return stdout.Bytes(), cmdError(vtyshCmd, string(stdout.String()), fmt.Errorf("%s: %s", err, strings.Replace(stderr.String(), "\n", " ", -1)))
```

Comment:
`cmdError` is only used here. So you could merge this one-line function.

Reply:
Done